### PR TITLE
loc fixes from CTI test pass

### PR DESCRIPTION
**Pull request by gcampbell-msft**
Contains fixes for some items that weren't localized.

### CODE REVIEW

**Comment by gcampbell-msft** [comment]
@snehara99 Ping for review.

**Comment by gcampbell-msft** [comment]
@snehara99 @benmcmorran ping for review.

File: src/drivers/cmakeDriver.ts [hunk 1]
```diff
@@ -1837,7 +1837,9 @@ export abstract class CMakeDriver implements vscode.Disposable {
         testPreset: preset.TestPreset | null,
         preferredGenerators: CMakeGenerator[]) {
         this._useCMakePresets = useCMakePresets;
-        log.debug(`Initializating base driver using ${useCMakePresets ? 'preset' : 'kit'}`);
+        const initBaseDriverWithPresetLoc = localize("init.driver.using.preset", "Initializing base driver using preset");
+        const initBaseDriverWithKitLoc = localize("init.driver.using.kit", "Initializing base driver using kit");
+        log.debug(`${useCMakePresets ? initBaseDriverWithPresetLoc : initBaseDriverWithKitLoc}`);
```
Comment (benmcmorran) [inline src/drivers/cmakeDriver.ts:1842]: Not asking for a change, but is the expectation in this extension that all log messages in are localized (even at high logging levels like debug which are probably intended for extension developers)?

**Review by benmcmorran — APPROVED** [review]

File: src/drivers/cmakeDriver.ts (hunk 1 above)
Comment (gcampbell-msft) [reply src/drivers/cmakeDriver.ts:1842]: Chatted offline. Yes this is the expectation since it's open source, it could be useful for those not in english.

**Pull request merged**